openapi.yaml

```diff
@@ -877,6 +877,85 @@ paths:
         - lang: CLI
           source: >
             linode-cli account invoice-items 123
+  /account/logins:
+    x-linode-cli-command: account
+    get:
+      tags:
+      - Account
+      summary: List All User Logins
+      description: >
+        Returns a collection of successful logins for all users on the account during the last
+        90 days. This command can only be accessed by the unrestricted users of an account.
+      operationId: getAccountLogins
+      x-linode-cli-action: logins-list
+      security:
+      - personalAccessToken: []
+      - oauth:
+        - account:read_only
+      responses:
+        '200':
+          description: >
+            A collection of successful logins for all users on the account during the last
+            90 days.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/Login'
+        default:
+          $ref: '#/components/responses/ErrorResponse'
+      x-code-samples:
+      - lang: Shell
+        source: >
+          curl -H "Authorization: Bearer $TOKEN" \
+              https://api.linode.com/v4/account/logins
+      - lang: CLI
+        source: >
+          linode-cli account logins-list
+  /account/logins/{loginId}:
+    parameters:
+    - name: loginId
+      in: path
+      description: The ID of the login object to access.
+      required: true
+      schema:
+        type: integer
+    x-linode-cli-command: account
+    get:
+      tags:
+      - Account
+      summary: View Login
+      description: >
+        Returns a login object that displays information about a successful
+        login from an arbitrary user on the account. This command can only be
```

Comment:
absolute nitpick here. "Arbitrary" user sounds weird to me, almost implying that the user isn't directly tied to the login object identified by the id.

It's a non-breaking thought, so if you feel like it stands well as-is then I'm all for it.

Reply:
I see what you mean... maybe an alternative could be:

Returns a Login object that displays information about a successful login. The logins that can be viewed can be for any user on the account, and are not restricted to only the logins of the user that is accessing this API endpoint. This command can only be accessed by the unrestricted users of the account.

Reply:
that works for me